packages/ember-self-focused/addon/components/self-focused.js

```diff
@@ -1,19 +1,28 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
+import { getOwner } from '@ember/application';
+import { computed } from '@ember/object';
 import layout from '../templates/components/self-focused';
 
 export default Component.extend({
   layout,
   focusManager: service(),
   classNames: ['self-focused'],
+  fastboot: computed(function() {
+    return getOwner(this).lookup('service:fastboot');
+  }),
 
   didInsertElement() {
     this._super(...arguments);
-    this.get('focusManager').didInsertElement(this.element);
+    if (!this.get('fastboot.isFastBoot')) {
```

Comment:
`didInsertElement` **should never** be called in fastboot, we shouldn't need to guard at all here...

Reply:
Done!